apisix/init.lua

```diff
@@ -1021,6 +1021,7 @@ function _M.stream_init_worker()
     plugin.init_worker()
     xrpc.init_worker()
     router.stream_init_worker()
+    require("apisix.http.service").init_worker()
```

Comment:
Can the `service.lua` be moved out of the `http` directory?

Reply:
no, obviously service is designed for http